runner/local/simple.go

```diff
@@ -13,22 +13,34 @@ import (
 
 func NewSimpleRunner(exec execer.Execer, checkouter snapshot.Checkouter, outputCreator runner.OutputCreator) runner.Runner {
```

Comment:
My natural inclination was that the caller took ownership of the channel: 'I need a runner and want it to report back to me when it is available'... this whole reporting back channel design has the following weaknesses: 
-the runner is passed in to queueingRunner and queueingRunner assumes that the runner initially is avaialable
- also, the isAvailable concept breaks down if/when the runner becomes a service - possibly shared by multiple callers (do we ever envision this architecture?).  

A stronger solution (IMHO) would be to simply send the Run() request and let the runner reject it if it is not available.  I think I started out with this approach but after conversations with Dan and in my eagerness to try using channels as mutex I may have gone overboard!  I think going back to this design (Run() rejects when not available - which is what runner does already) would actually simplify the QueueingRunner code.